content/zh/news/releases/1.6.x/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: 1.6.x 版本
```

Comment:
Normally, we don't maintain things that have reached EOL. Please see the supported release page at https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases to see the list of versions we are maintaining

Reply:
😭 Unfortunately, At before, I didn't notice the information here.